### PR TITLE
add ali-oss type override

### PR DESCRIPTION
**Pull request by running-grass**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <<https://github.com/ali-sdk/ali-oss#copyname-sourcename-sourcebucket-options>>
- [x] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## ali-oss ([<kbd>unpkg</kbd>](https://unpkg.com/browse/ali-oss@latest/))
was missing the following properties:
1. initOptions
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 42d5d18cb18cb28b237d05898df989c6b39511b4
</p>

File: types/ali-oss/index.d.ts [hunk 1]
```diff
@@ -1029,7 +1029,8 @@ declare class OSS {
     /**
      * Copy an object from sourceName to name.
      */
-    copy(name: string, sourceName: string, options?: OSS.CopyObjectOptions): Promise<OSS.CopyAndPutMetaResult>;
+     copy(name: string, sourceName: string, options?: OSS.CopyObjectOptions): Promise<OSS.CopyAndPutMetaResult>;
+     copy(name: string, sourceName: string, sourceBucket?: string, options?: OSS.CopyObjectOptions): Promise<OSS.CopyAndPutMetaResult>;
```
Comment (MysteryBlokHed) [inline types/ali-oss/index.d.ts:1033]: Could you unindent these lines? Keeps the formatting consistent 😄

```suggestion
    copy(name: string, sourceName: string, options?: OSS.CopyObjectOptions): Promise<OSS.CopyAndPutMetaResult>;
    copy(name: string, sourceName: string, sourceBucket?: string, options?: OSS.CopyObjectOptions): Promise<OSS.CopyAndPutMetaResult>;
```

**Review by MysteryBlokHed — CHANGES_REQUESTED** [review]

**Review by MysteryBlokHed — APPROVED** [review]
LGTM!

**Comment by running-grass** [comment]
Ready to merge

**Pull request merged**